starfyre/compiler.py

```diff
@@ -32,10 +32,28 @@ def resolve_css_import(css_file_name, working_directory):
         for line in import_file.readlines():
             css_content.append(line)
 
-    return css_content           
+    return css_content
 
 
-def parse(fyre_file_name): 
+def check_import_line(line, project_dir):
```

Comment:
Does this function only match `from {module} import module`?

Reply:
Yes, it does.

Reply:
@DeleMike but what about `import {module}` statements?

Reply:
First of all, let me state that the current import styles that would work for the `test_application` are:
`from test_application.parent import parent` and `from ..store import store`

So if you say something like `import {module}`, do you mean to ask if something like `import parent` or `import store` is working?

Reply:
@DeleMike yess

Reply:
Oh, thank you.
Ideally, It should work but I don't think Starfyre has the opportunity for this style of writing. 

I will look at this and get back to you. Ty!

Reply:
@DeleMike , it definitely does. e.g.

```python
import pprint

def fx():
  pprint.pprint("This is the task")

...fx used somewhere else
```

Reply:
Hello @sansyrox,

I have some updates:
So this is a typical import style in Starfyre:
```
from test_application.parent import parent
from ..store import store
```

But if I change it to this:
```
from test_application.parent import parent
import store
```

It throws this error:
```
Traceback (most recent call last):
  File "/home/xxxx/software_eng/starfyre/test_application/build/__main__.py", line 40, in generate_pages
    module = importlib.import_module(module_name)
  File "/usr/local/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/xxxx/software_eng/starfyre/test_application/build/pages/__init__.py", line 2, in <module>
    import store
ModuleNotFoundError: No module named 'store'
```
To resolve this:
I tried this:
I tried to first look at the `build_main_file_content.py` file and it was difficult to resolve it from there. Then I looked at trying to resolve it via the `compiler.py`. My approach is that if we see an `import store` we should change it to an absolute style of import. That would work! 

However, we might want to import other things other than our custom components. Recall, that we can combine `Python` and `fyre` syntax in one file. For example;
In the `parent.fyre`, we imported the `requests` package and if we are to parse this file in the compiler, the `import requests` will be read and considered as an import statement that should be modified in order to resolve the `custom components`.

These are my observations so far. What should we do now?

Reply:
we should not convert the `import {module}` statements to `from {module} import {module}` . This changes the entire import syntax. We should keep the regular imports as is.

Reply:
> We should keep the regular imports as is.

Hello @sansyrox 
Yes, you are right. 

Do you have any recommendations to make this import style function correctly?

Should we keep using only the absolute and relative styles for the time being?

Reply:
Hey @DeleMike 👋 

> Should we keep using only the absolute and relative styles for the time being?

Yes, let's do this 😄 

Also, can you please open an issue about absolute imports?

Reply:
Hello @sansyrox, sorry for the delayed response. 

Thank you. I will do just that!